acoustid/future/fpindex/client.py

```
@@ -0,0 +1,396 @@
+        try:
+            async with self.session.request(method, url, **kwargs) as response:
+                status = response.status
+
+                if expected_status and status not in expected_status:
+                    body = await response.text()
+                    logger.error(
+                        "Unexpected status %s for %s %s: %s", status, method, url, body
+                    )
+                    raise FingerprintIndexClientError(
+                        f"Unexpected status {status}: {body}"
+                    )
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)